lib/modules/newCommentCount.js

```diff
@@ -166,9 +171,13 @@ addModule('newCommentCount', function(module, moduleID) {
 				$thisROW.find('td:first-of-type').append(thisTrash);
 				if (isSubscribed) {
 					$thisROW.find('td:last-of-type').append(thisRenewButton).append(' ');
-					$thisROW.find('td:last-of-type').append(thisUnsubButton);
+					if (!module.options.showSubscribeButton.value){
```

Comment:
This logic seems the opposite of what you want: if the "show button" option is not turned on, add the button? I would have expected "if show button is enabled, add the button", i.e. remove `!`